percy/render/_renderer.py

```diff
@@ -252,21 +270,31 @@ def expand_compiler(lang):
                 )
         elif renderer_type == RendererType.CONDA:
             if not has_conda_build:
-                raise YAMLRenderFailure(recipe_dir, message="conda build unavailable")
+                raise YAMLRenderFailure(recipe_file, message="conda build unavailable")
             platform, arch = selector_dict.get("subdir").split("-")
             rendered = api.render(
-                recipe_dir,
+                recipe_file,
                 config=Config(
                     platform=platform,
                     arch=arch,
                 ),
                 variants=selector_dict,
             )
             return rendered[0][0].meta
-        elif renderer_type == RendererType.PERCY:
-            parser = RecipeParser(meta_yaml)
-            return parser.render_to_object()
+        elif renderer_type == RendererType.CRM:
+            rendered = RecipeParserDeps(meta_yaml)
+            return rendered
+        elif renderer_type == RendererType.RUAMEL_JINJA:
+            yaml_jinja = YAML(typ="jinja2")
+            yaml_jinja.indent(mapping=2, sequence=4, offset=2)
+            yaml_jinja.preserve_quotes = True
+            yaml_jinja.allow_duplicate_keys = True
+            yaml_jinja.width = 1000
```

Comment:
```suggestion
            yaml_jinja.width = 2048
```
I would recommend 2048 because I had issues with some recipes where source urls where too long.

Reply:
Could you check if these attributes are also useful?
```
    yaml.sort_keys = True
    yaml.explicit_start = False
```
When I experimented with updating recipes, I found that they are required for some cases, but, honestly, I forgot the reason

Reply:
addressed the width. I do not want to sort the keys. explicit_start true adds the whole content as a list item